packages/cli/CommandLineHelp.md

```
@@ -381,6 +383,37 @@ Fetch raw Prometheus metrics from the running indexer's /metrics endpoint
+
+Earliest USDC transfers on Base:
+
+``` envio data block.number log.srcAddress log.transactionHash \ --chain=base \ --where='{ block: { number: { _gte: 0 } }, log: { srcAddress: "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913" }, }' ```
+
+Current archive height:
+
+``` envio data knownHeight --chain=arbitrum-one ```
+
+**Usage:** `envio data [OPTIONS] --chain <CHAIN> [FIELD]...`
+
+###### **Arguments:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)